src/librustc_session/options.rs

```diff
@@ -950,6 +961,12 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
          (such as entering an empty infinite loop) by inserting llvm.sideeffect"),
     deduplicate_diagnostics: Option<bool> = (None, parse_opt_bool, [UNTRACKED],
         "deduplicate identical diagnostics"),
+    // The control_flow_guard option needs to encode three possibilities:
```

Comment:
Nit: a better place for this comment would be on the `CFGuard` type or on the function that parses it, rather than here.

Reply:
Fixed